streaming/base/converters/dataframe_to_mds.py

```diff
@@ -264,6 +264,10 @@ def write_mds(iterator: Iterable):
     if 'out' not in mds_kwargs:
         raise ValueError(f'`out` and `columns` need to be specified in `mds_kwargs`')
 
+    if 'compression' not in mds_kwargs:
+        logger.info('Defaulting to zstd compression')
+        mds_kwargs['compression'] = 'zstd'
```

Comment:
This may be very well true for the LLM dataset, but it may not be that useful for the Image-based models since JPEG/PNG is already compressed!!! and you won't see much of the benefits in terms of storage space.

Reply:
I agree, that's the type of usage I was thinking of too. I was wondering how much good vs bad this default would do.

For example, in the common case of tokens or text, this probably reduces the size by 3x.
In the case of images, it won't compress and adds a bit of overhead.

It seemed like it would more often help than hurt so I had wanted to establish it as a default, but maybe not worth a behavior change. Not sure.

Reply:
Performance wise this is perfectly fine even with images. I'm good with merging